src/net.rs

```diff
@@ -312,7 +331,13 @@ impl Packet {
         coordinator_public_key: &ecdsa::PublicKey,
     ) -> bool {
         match &self.msg {
-            Message::DkgBegin(msg) | Message::DkgPrivateBegin(msg) => {
+            Message::DkgBegin(msg) => {
+                if !msg.verify(&self.sig, coordinator_public_key) {
+                    warn!("Received a DkgPrivateBegin message with an invalid signature.");
```

Comment:
Copy paste error: Should be "Received a DkgBegin message..."

Reply:
Done.